Controller/HealthCheckController.php

```diff
@@ -17,15 +19,25 @@ class HealthCheckController
     protected $template;
 
     /**
-     * @param RunnerManager $runnerManager
-     * @param PathHelper    $pathHelper
-     * @param               $template
+     * @var Psr\Log\LoggerInterface
      */
-    public function __construct(RunnerManager $runnerManager, PathHelper $pathHelper, $template)
+    protected $log;
+
+    /**
+     * @param RunnerManager        $runnerManager
+     * @param PathHelper           $pathHelper
+     * @param string               $template
+     * @param LoggerInterface|null $log
+     */
+    public function __construct(RunnerManager $runnerManager, PathHelper $pathHelper, $template, LoggerInterface $log = null)
     {
         $this->runnerManager = $runnerManager;
         $this->pathHelper = $pathHelper;
         $this->template = $template;
+        if ($log === null) {
+            $log = new NullLogger();
```

Comment:
OK, I'll remove the NullLogger, and require a valid logger to be injected. Users can can replace it with another Logger, Null, or otherwise.